src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -136,6 +136,24 @@ protected void init(DerValue encoding, int req_type)
                         "encoding tag is " +
                         encoding.getTag() +
                         " req type is " + req_type);
+
+                System.out.println(">>> KDCRep: Message in bytes is =>");
+                byte[] dataBytes = encoding.getDataBytes();
+                StringBuilder sb = new StringBuilder();
+                for (int i = 0; i < dataBytes.length; i++) {
+                    if ((i % 16) == 0) {
+                        sb.append(String.format("%06X ", i));
+                    }
+                    byte b = dataBytes[i];
+                    sb.append(String.format("%02X ", b));
```

Comment:
nit: this produces lines which end with a space; suggest:
```java
                        sb.append(String.format("%06X", i));
                    }
                    byte b = dataBytes[i];
                    sb.append(String.format(" %02X", b));
```

Reply:
Fixed.